### PR TITLE
Prac 09 feedback

### PR DESCRIPTION
**Pull request by Riston-gitting-gud**
Hello, please take a look at my code and point out any mistakes, thank you!
@MikeLeongZhiFai
@AdriantoHartono
@JevanChandra

### CODE REVIEW

**Review by MikeLeongZhiFai — COMMENTED** [review]
Code is well written but putting a MENU will be better! I took reference from your code too as I am struggling. Thanks!!!

**Review by AdriantoHartono — COMMENTED** [review]
Great code, already written neatly in each function👍

File: prac_09/band.py
Comment (JevanChandra) [inline prac_09/band.py:1]: Nice work

Comment (Riston-gitting-gud) [reply prac_09/band.py:1]: Thanks!

**Pull request merged**